src/app/common/duck/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from "history";
+
+export const history = createBrowserHistory();
```

Comment:
Why not to `export default`?

Reply:
what if there are other utils we want to add to the utils file?

Reply:
Oh, yeah, my fault, I still have in my head old structure.